### DISCUSSION TITLE
missing popup windows - example in gnome-control-center

### DISCUSSION THREAD

**Discussion by simoncasassus** [regolith-linux/regolith-desktop#1033] [General]
I am running Regolith 3.2 on Ubuntu 24.04, it is all going smoothly except for missing pop-up windows, perhaps related to the same problem. For example, the Users gui in gnome-control-center prompts to Unlock, but the window does not pop up, for which I need to revert temporarily to standard Gnome wm.  I get similar issues when connecting to public wifi networks that require a dialog.

**Comment by holtzermann17** [comment]
Here's an example showing that I can't easily adjust the volume levels via `gnome-control-center`. 

```
Ubuntu 24.04 LTS
REGOLITH_VERSION=3.2
XDG_SESSION_TYPE=x11
DESKTOP_SESSION=regolith-x11
```

![Screenshot from 2024-09-02 09-53-37](https://github.com/user-attachments/assets/1fdc7584-666f-4d8a-b626-73efda6cb52e)

**Reply by kgilmer** [reply]
![image](https://github.com/user-attachments/assets/4bd0bf19-e53f-4104-86ec-d61784cfeb91)

Interesting problem.  Nothing obvious comes to mind as a root cause to this; `gnome-control-center` is unmodified from upstream.  I'm seeing the dialog populated both in `wayland` and `x11` sesssions on my Debian Testing / Regolith 3.2 machine.

**Comment by holtzermann17** [comment]
Maybe the specific issue I saw is more on the `gnome-control-center` side; as of today, it's not creating a pop-up at all, i.e., now I see this:
![Screenshot from 2024-09-04 13-14-52](https://github.com/user-attachments/assets/54ed208c-9604-4b5c-910b-3215c8e8e1cc)
